### ISSUE TITLE
Warning: Operand of null-aware operation '??'

### ISSUE THREAD

**Issue by JAICHANGPARK** [entronad/flutter_echarts#119]
I had some warning msg on 2.0.0 version 

```
Warning: Operand of null-aware operation '??' has type 'String' which excludes null.
        (value, element) => (value ?? '') + '\n' + (element ?? '')
```

**Comment by singhtaranjeet** [comment]
it is stating that value is of type `String` which can't be **null** so, you don't have to use the `?? ''` for String but you can use `?? ''` for **String?**
please do look the official doc: [https://dart.dev/null-safety](url)

**Comment by brylie** [comment]
I am getting this warning when trying out the examples. However, the example seems to work.

**Comment by entronad** [comment]
This is a redundant null-check lint warning. It is remained from the old version without null-safety.
It is harmless. I will check and remove them thoroughly when I have time.

**Comment by brylie** [comment]
Thank you @entronad :-)

**Comment by entronad** [comment]
Fixed in v2.1.0

**Comment by JAICHANGPARK** [comment]
@entronad Thank you for supporting
